description.md

```diff
@@ -13,6 +13,46 @@ Each section details how a specific type of content is to be developed, stored a
 
 ### Media
 
+The `media` content type refers to audio, visual or audiovisual educational materials.
+Images, videos, recordings, educational video games represent examples of such content.
+
+The media content type represents an easily digestible format for presenting information.
+As such, media is typically embbeded within the text that is destined reading, slides, demos, practice items and projects.
+
+When developing `media` items, it is important to follow the guidelines below:
+
+- use an open source or free development tool.
+- store both the output (the actual image, video etc.) and the editable building blocks.
+
+#### Images
+
+For the creation of images and diagrams we use [draw.io](draw.io).
+`draw.io` is a free, easy to use tool which is available in the browser.
+The resulting images can be exported in different formats, such as PDF, PNG or SVG.
+When storing images, it is important that both the source file (with the extension `drawio` or `svg`) and the immutable output (PDF, PNG, JPG) are stored.
+Note that other tools for the generation of images may be used as long as they are free and provide the source file.
+
+#### Audio-video recordings
+
+Audio or video content is by its nature hard to modify.
```

Comment:
I would do a classification of audio-video content and correlate it with use cases. Otherwise, it simply appears as a listing of potential approaches, but the reader wouldn't know what to use where.

I would add:
- Animations: succession of images that show a process, a set of steps - these can be created with an image software and then an image-tying software such as ffmpeg
- Screencasts: someone records their screen, it requires a screen recording software - give examples, this typically screencasts applications (terminal, browser, simulators)
- Actual recordings: a live recording with a camera of someone giving a talk or detailing a concept; this can also be, in certain cases, an audio recording only

I would then add sections on each of those and detail the actual methodology for producing these types of contents, together with sample tools they would use for them.

Reply:
Either here, when presenting the media content types, or in each section, provide use cases for each of those. The reader should have answers to questions:
- What should I do if I want to do X (with respect to media content)? (get from use case to tool)
- What should I use this media content type for? (get from tool to use case)